.env

```diff
@@ -0,0 +1,4 @@
+TEMPLATE=False
```

Comment:
I was saying about this file by the way :(

Reply:
Oh I missed that. Apologies. Will rectify it right now.